source/agora/test/Base.d

```diff
@@ -921,6 +919,7 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
             retry_delay : test_conf.retry_delay, // msecs
             max_retries : test_conf.max_retries,
             timeout : test_conf.timeout,
+            validator_cycle : cons_params.ValidatorCycle,
```

Comment:
This probably will go in the future, in favor of adding direct support in `TestConf`.
I'm also considering making 20 the default as we're likely to want to test cycles more often than not.